roles/junit2json/tasks/convert.yml

```
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file for redhatci.ocp.junit2json
+# task list: convert.yml converts xml into JSON
+
+- name: Read file content
+  ansible.builtin.set_fact:
+    junit2json_xml_report_content: "{{ lookup('ansible.builtin.file', item) }}"
+
+- name: Update junit2json_result_data junit2json_do_merge=true
+  ansible.builtin.set_fact:
+    junit2json_result_data: "{{ _junit2json_merged_xml.stdout if junit2json_do_merge else junit2json_xml_report_content }}"
```

nit, in cases like this use ternary:
```suggestion
    junit2json_result_data: "{{ junit2json_do_merge | ternary(_junit2json_merged_xml.stdout, junit2json_xml_report_content) }}"
```